pkg/key/app.go

```diff
@@ -142,6 +150,10 @@ func IsManagedByFlux(customResource v1alpha1.App, projectName string) bool {
 	return customResource.Labels[label.ManagedBy] == "flux"
 }
 
+func IsManagedInOrg(customResource v1alpha1.App) bool {
+	return strings.HasPrefix(customResource.ObjectMeta.Namespace, "org-")
```

Comment:
This check feels a bit simplified, not sure if that's ok. Is `org-` suffix something I can relay on here, or should I refer to the Organization CR and implement something more sophisticated?

Reply:
Ok, I've seen it has always been like that, so I guess we can relay on that https://github.com/giantswarm/organization-operator/blob/af6b447437d2bd3233957023f12a222eabaf09eb/service/controller/resource/organization/resource.go#L18

Reply:
You could also list Organization CRs and see if any have a `.status.namespace` (https://github.com/giantswarm/apiextensions/blob/master/pkg/apis/security/v1alpha1/organization_types.go#L32) equal to the app namespace. I don't think it's worth the effort though.

Reply:
Yes I don't think checking the organization CRs is worth it and it would make the dependencies more complex.